static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -163,7 +158,7 @@ const showBanner = (params: EngagementBannerParams): void => {
     }).show(renderedBanner);
 
     if (messageShown) {
-        ['INSERT', 'VIEW'].forEach(action => {
+        [('INSERT', 'VIEW')].forEach(action => {
```

Comment:
pretty sure you don't want the round brackets here - you end up with an array only containing `'VIEW'` since what's inside the brackets are evaluated as one expression and the comma operator returns the rightmost value